src/walletClient/ClientMethods.js

```diff
@@ -1371,6 +1371,55 @@ exports.GiftClaimStatus = async function({marketplaceParams, confirmationId, gif
   }
 };
 
+/**
+ * Return status of the specified entitlement claim
+ *
+ * @methodGroup Status
+ * @namedParams
+ * @param {Object} marketplaceParams - Parameters of the marketplace
+ * @param {string=} confirmationId - The confirmation ID of the entitlement
+ *
+ * @returns {Promise<Object>} - The mint status of the gift claim
+ */
+exports.EntitlementClaimStatus = async function({marketplaceParams, purchaseId}) {
```

Comment:
can choose either .EntitlementClaimStatus or ..EntitlementMint[ing]Status for this and the below